src/filter.ts

```diff
@@ -100,11 +81,7 @@ export class PropertyFilter extends EntityFilter implements IFilter {
    */
   // eslint-disable-next-line
   toProto(): any {
-    const value = new PropertyFilter(
-      this.name,
-      this.op,
-      this.val
-    ).encodedValue();
+    const value = entity.encodeValue(this.val, this.name);
```

Comment:
You can also `value: entity.encodeValue(this.val, this.name)` inline. Not sure what the preferred style is.

Reply:
Yes. This can be even more compact.